src/components/time-picker/time-picker.html

```diff
@@ -0,0 +1,22 @@
+<div class="bx--form-item">
+  <label for="time-picker-1" class="bx--label">Label</label>
+  <div id="time-picker-1" data-time-picker class="bx--time-picker">
+    <div data-time-picker-input class="bx--time-picker__input">
+      <input data-time-picker-input-field type="number" class="bx--time-picker__input-field bx--time-picker__input-field--hours" placeholder="hh" step="1" min="0" />
+      <span class="bx--time-picker__separator">:</span>
```

Comment:
Time separator depends on locales. Given that, I think it'd be easier for us to let user input the separator - Meaning, one input rather than two inputs.